tests/components/bangolufsen/const.py

```diff
@@ -0,0 +1,189 @@
+"""Constants used for testing the bangolufsen integration."""
+
+from multiprocessing.pool import AsyncResult, Pool
+from unittest.mock import Mock
+
+from mozart_api.exceptions import ApiException
+from mozart_api.models import BeolinkPeer, VolumeLevel, VolumeSettings
+from urllib3.exceptions import MaxRetryError, NewConnectionError
+
+from homeassistant.components.bangolufsen.const import (
+    ATTR_FRIENDLY_NAME,
+    ATTR_ITEM_NUMBER,
+    ATTR_SERIAL_NUMBER,
+    ATTR_TYPE_NUMBER,
+    CONF_BEOLINK_JID,
+    CONF_DEFAULT_VOLUME,
+    CONF_MAX_VOLUME,
+    CONF_VOLUME_STEP,
+)
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.const import CONF_HOST, CONF_MODEL, CONF_NAME
+
+
+class TestConstants:
+    """Constants for general testing."""
+
+    TEST_HOST = "192.168.0.1"
+    TEST_HOST_INVALID = "192.168.0"
+    TEST_MODEL_BALANCE = "Beosound Balance"
+    TEST_MODEL_THEATRE = "Beosound Theatre"
+    TEST_MODEL_LEVEL = "Beosound Level"
+    TEST_SERIAL_NUMBER = "11111111"
+    TEST_NAME = f"{TEST_MODEL_BALANCE}-{TEST_SERIAL_NUMBER}"
+    TEST_FRIENDLY_NAME = "Living room Balance"
+    TEST_TYPE_NUMBER = "1111"
+    TEST_ITEM_NUMBER = "1111111"
+    TEST_JID_1 = f"{TEST_TYPE_NUMBER}.{TEST_ITEM_NUMBER}.{TEST_SERIAL_NUMBER}@products.bang-olufsen.com"
+    TEST_JID_2 = f"{2222}.{2222222}.{22222222}@products.bang-olufsen.com"
+    TEST_JID_3 = f"{3333}.{3333333}.{33333333}@products.bang-olufsen.com"
+    TEST_DEFAULT_VOLUME = 40
+    TEST_MAX_VOLUME = 100
+    TEST_VOLUME_STEP = 5
+
+
+class TestConstantsConfigFlow(TestConstants):
```

Comment:
Please remove the classes from this file. You can create different test configurations using test fixtures.

Reply:
I've revamped the testing to (hopefully) properly use fixtures